.package-version

```diff
@@ -0,0 +1 @@
+8.14.0
```

Comment:
I left it at 8.14.0 on purpose to test that the new automation works and updates it to 8.15.0 the next day.